src/main/java/hudson/plugins/logparser/LogParserAction.java

```diff
@@ -31,13 +32,17 @@
 
 public class LogParserAction implements Action {
 
-    final private AbstractBuild<?, ?> build;
+    final private Run<?, ?> build;
     final private LogParserResult result;
 
     private static String urlName = "parsed_console";
 
-    public LogParserAction(final AbstractBuild<?, ?> build,
-            final LogParserResult result) {
+    @Deprecated
+    public LogParserAction(final AbstractBuild<?, ?> build, final LogParserResult result) {
+        this((Run<?, ?>) build, result);
+    }
+
+    public LogParserAction(final Run<?, ?> build, final LogParserResult result) {

```

Comment:
Are both of these args mandatory? Maybe mark them with `@Nonnull` / `@Checkfornull` etc.